images.py

```diff
@@ -55,16 +58,38 @@ def _server_query(url, q):
     return doc
 
 
-# The Custom Search identifier
-_CX = "001858240983628375092:9aogptqla5e"
 # The Google API identifier (you must obtain your own key if you want to use this code)
 _API_KEY = ""
+_API_KEY_PATH = "resources/GoogleServerKey.txt"
```

Comment:
Use `os.path.join()` here (don't forget Windows! :-) )

Reply:
... and actually you probably need:
```
_PATH = os.path.dirname(__file__) or "."
_API_KEY_PATH = os.path.join(_PATH, "resources", "GoogleServerKey)
```

Reply:
How about using pathlib? Do we need to support Python 3.3 and earlier?

Reply:
Nope, Python >= 3.4 is the requirement